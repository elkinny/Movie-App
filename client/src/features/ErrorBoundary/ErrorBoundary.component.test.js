import ErrorBoundaryComponent from './ErrorBoundary.component';
import React from 'react';
import { mount } from './../../test/enzymeSetup.js';

describe('test: ', () => {
  it('ErrorBoundaryComponent to mathc snapshot', () => {
    const props = {
      errorName: 'errorName',
      componentStack: 'componentStack',
    };
    const component = mount(<ErrorBoundaryComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
