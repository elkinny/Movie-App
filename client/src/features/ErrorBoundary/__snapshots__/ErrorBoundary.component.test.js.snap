// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test:  ErrorBoundaryComponent to mathc snapshot 1`] = `
<ErrorBoundaryComponent
  componentStack="componentStack"
  errorName="errorName"
>
  <div
    className="error"
  >
    <h1
      className="error__title"
    >
      Oops! An error occured!
    </h1>
    <h2
      className="error__subtitle"
    >
      Here’s what we know…
    </h2>
    <p
      className="error__name"
    >
      <span
        className="error__label"
      >
        Error:
      </span>
      <span>
        errorName
      </span>
    </p>
    <article
      className="error__stacktrace"
    >
      <h3
        className="error__label"
      >
        Stacktrace:
      </h3>
      <p
        className="error__stacktrace-item"
        key="trace0"
      >
        componentStack
      </p>
    </article>
  </div>
</ErrorBoundaryComponent>
`;
